fast_matched_filter/fast_matched_filter.py

```diff
@@ -272,6 +315,14 @@ def matched_filter(
     elif arch == "precise" and ~network_sum:
         args = args + (normalize,)
         _libCPU.matched_filter_precise_no_sum(*args)
+    elif arch == "variable_precise" and network_sum:
+        args = args + (normalize,)
+        _libCPU.matched_filter_variable_precise(*args)
+    elif arch == "variable_precise" and ~network_sum:
+        #args = args + (normalize,)
+        #_libCPU.matched_filter_variable_precise_no_sum(*args)
+        print("Not yet implemented!")
+        return
```

Comment:
```suggestion
        raise NotImplementedError("Variable precise without network sum is not yet implemented.")
```
A print statement might be hard to spot for the user.